app/src/main/java/co/nimblehq/di/modules/FragmentComponentsModule.kt

```diff
@@ -14,3 +17,11 @@ abstract class NavigatorModule {
     @Binds
     abstract fun mainNavigator(mainNavigator: MainNavigatorImpl): MainNavigator
 }
+
+@InstallIn(FragmentComponent::class)
+@Module
+class RxPermissionsModule {
+
+    @Provides
+    fun rxPermission(fragment: Fragment): RxPermissions = RxPermissions(fragment)
```

Comment:
No biggie: `rxPermissions`

Same goes for `lateinit var rxPermissions` & `rxpermissions_version`